c9y-test/paralell_test.cpp

```diff
@@ -122,14 +230,22 @@ std::vector<std::string> tokenize(const std::string_view str, const std::string_
         else
         {
             gib = str.substr(start, end - start);
-            start = end + 1;
+            start = str.find_first_not_of(delimiters, end);
         }
         gibs.push_back(gib);
     }
 
     return gibs;
 }
 
+TEST(parallel, tokenize)
```

Comment:
Although this test has it's merits, put it into a different suite; because this is not actually a parallel algorithm.